modules/ROOT/pages/mule-application-about.adoc

```diff
@@ -33,6 +35,8 @@ For example, the Salesforce Connector provides components that let you use the S
 .The Salesforce connector and its components listed in Anypoint Studio
```

Comment:
Again, why confuse people by using "component" generically?

Reply:
I went ahead and attempted to clarify the terminology it because it's otherwise out of sync with _every_ other doc that talks about this stuff in our doc set.